CHANGELOG.md

```diff
@@ -69,9 +69,16 @@
 
 ## Unreleased
 
+### Breaking Changes
+
+#### Plugins
+
+- **JWT**: JWT plugin now denies a request that has different tokens in the jwt token search locations.
```

Comment:
```suggestion
- **JWT**: JWT plugin now denies a request that has different JWTs in the HTTP request header and Query parameter.
```

Reply:
There are three search locations, which are querystring(query parameter), header, and cookie.